src/test/java/uk/gov/hmcts/reform/professionalapi/service/OrganisationServiceImplTest.java

```diff
@@ -232,7 +230,6 @@ public void testSavesAnOrganisation() {
         prdEnums.add(new PrdEnum(new PrdEnumId(0, "SIDAM_ROLE"), "pui-user-manager", "SIDAM_ROLE"));
         prdEnums.add(new PrdEnum(new PrdEnumId(4, "ADMIN_ROLE"), "organisation-admin", "ADMIN_ROLE"));
         prdEnums.add(new PrdEnum(new PrdEnumId(10, "JURISD_ID"), "PROBATE", "PROBATE"));
-        prdEnums.add(new PrdEnum(new PrdEnumId(10, "JURISD_ID"), "PROBATE", "PROBATE"));

```

Comment:
Why was this removed? I believe this was required to satisfy a mutation scenario...

Reply:
I believe that this line is a duplication of the line above

Reply:
😂I come across like a goon, but ah cool, git didn't make that look very clear